<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouteCalcDialog</class>
 <widget class="QDialog" name="RouteCalcDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Flight Plan Calculation</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="labelRouteCalcHeader">
       <property name="frameShape">
        <enum>QFrame::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>5</number>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_40">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QRadioButton" name="radioButtonRouteCalcFull">
        <property name="toolTip">
         <string>Calculates a flight plan between departure and destination.
Keeps procedures.</string>
        </property>
        <property name="statusTip">
         <string>Calculates a flight plan between departure and destination</string>
        </property>
        <property name="text">
         <string>Calculate from &amp;departure to destination</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radioButtonRouteCalcSelection">
        <property name="toolTip">
         <string>Calculates a route between the selected legs in the flight plan table</string>
        </property>
        <property name="statusTip">
         <string>Calculates a route between the selected legs in the flight plan table</string>
        </property>
        <property name="text">
         <string>Calculate for &amp;selection in flight plan table</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QLabel" name="labelRouteCalcCruiseAltitude">
        <property name="text">
         <string>&amp;Cruise altitude for calculation:</string>
        </property>
        <property name="buddy">
         <cstring>spinBoxRouteCalcCruiseAltitude</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBoxRouteCalcCruiseAltitude">
        <property name="toolTip">
         <string>Cruise altitude that is used to select airways</string>
        </property>
        <property name="statusTip">
         <string>Cruise altitude that is used to select airways</string>
        </property>
        <property name="showGroupSeparator" stdset="0">
         <bool>true</bool>
        </property>
        <property name="suffix">
         <string> %alt%</string>
        </property>
        <property name="maximum">
         <number>100000</number>
        </property>
        <property name="singleStep">
         <number>500</number>
        </property>
        <property name="value">
         <number>10000</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRouteCalcAdjustAltitude">
        <property name="toolTip">
         <string>Adjust flight plan altitude using simplified east/west and IFR/VFR rules</string>
        </property>
        <property name="statusTip">
         <string>Adjust flight plan altitude using simplified east/west and IFR/VFR rules</string>
        </property>
        <property name="text">
         <string>A&amp;djust</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_6">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <property name="spacing">
       <number>2</number>
      </property>
      <item row="5" column="1" colspan="2">
       <widget class="QGroupBox" name="groupBoxRouteCalcAirwayPrefer">
        <property name="title">
         <string>Prefer:</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <property name="topMargin">
          <number>2</number>
         </property>
         <property name="rightMargin">
          <number>2</number>
         </property>
         <property name="bottomMargin">
          <number>2</number>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
         <item row="1" column="2">
          <widget class="QLabel" name="labelRouteCalcAirwayPreferWaypoint">
           <property name="text">
            <string>Direct</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelRouteCalcAirwayPreferAirway">
           <property name="text">
            <string>Airways</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelRouteCalcAirwayPreferWaypoint_4">
           <property name="text">
            <string>Both</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="3">
          <widget class="QSlider" name="horizontalSliderRouteCalcAirwayPref">
           <property name="toolTip">
            <string>Move the slider left to use airways only. This is the fastest calculation method.
Move the slider to the right to use direct waypoint connections only.
Every position in between uses airways and waypoints more or less and
allows to find a flight plan that can cross areas without an airway network or
unusable airways (e.g. due to altitude restrictions).</string>
           </property>
           <property name="maximum">
            <number>14</number>
           </property>
           <property name="pageStep">
            <number>1</number>
           </property>
           <property name="value">
            <number>7</number>
           </property>
           <property name="tracking">
            <bool>true</bool>
           </property>
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="invertedAppearance">
            <bool>false</bool>
           </property>
           <property name="invertedControls">
            <bool>false</bool>
           </property>
           <property name="tickPosition">
            <enum>QSlider::TicksBothSides</enum>
           </property>
           <property name="tickInterval">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item row="2" column="0" colspan="3">
          <widget class="QLabel" name="labelRouteCalcPreference">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Airways and tracks only.
Fastest calculation</string>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QRadioButton" name="radioButtonRouteCalcAirwayVictor">
        <property name="toolTip">
         <string>Limit route to low airways (Victor)</string>
        </property>
        <property name="statusTip">
         <string>Limit route to low airways (Victor)</string>
        </property>
        <property name="text">
         <string>&amp;Low airways</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="4" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="pushButtonRouteCalcTrackDownload">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string>Download NAT and PACOTS tracks now</string>
          </property>
          <property name="statusTip">
           <string>Download NAT and PACOTS tracks now</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../../littlenavmap.qrc">
            <normaloff>:/littlenavmap/resources/icons/airwaytrackdown.svg</normaloff>:/littlenavmap/resources/icons/airwaytrackdown.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="checkBoxRouteCalcAirwayTrack">
        <property name="toolTip">
         <string>Use downloaded NAT or PACOTS tracks.
Best track will be selected automatically.</string>
        </property>
        <property name="statusTip">
         <string>Use downloaded NAT or PACOTS tracks.</string>
        </property>
        <property name="text">
         <string>&amp;Use NAT or PACOTS tracks</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QCheckBox" name="checkBoxRouteCalcAirwayNoRnav">
        <property name="toolTip">
         <string>Do not use RNAV airways.
Disabled if a navdatabase does not allow it.</string>
        </property>
        <property name="statusTip">
         <string>Do not use RNAV airways</string>
        </property>
        <property name="text">
         <string>Avoid &amp;RNAV airways</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radioButtonRouteCalcAirway">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Calculate a flight plan using airways and waypoints</string>
        </property>
        <property name="statusTip">
         <string>Calculate a flight plan using airways and waypoints</string>
        </property>
        <property name="text">
         <string>&amp;Airways:</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QCheckBox" name="checkBoxRouteCalcRadioNdb">
        <property name="toolTip">
         <string>Include NDB stations in plan</string>
        </property>
        <property name="statusTip">
         <string>Include NDB stations in plan</string>
        </property>
        <property name="text">
         <string>Use &amp;NDB</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QRadioButton" name="radioButtonRouteCalcRadio">
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Calculate a flight plan using radio navaids like VOR and NDB</string>
        </property>
        <property name="statusTip">
         <string>Calculate a flight plan using radio navaids like VOR and NDB</string>
        </property>
        <property name="text">
         <string>&amp;Radio Navaids:</string>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup</string>
        </attribute>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QRadioButton" name="radioButtonRouteCalcAirwayJet">
        <property name="toolTip">
         <string>Limit route to high airways (Jet)</string>
        </property>
        <property name="statusTip">
         <string>Limit route to high airways (Jet)</string>
        </property>
        <property name="text">
         <string>&amp;High airways</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QRadioButton" name="radioButtonRouteCalcAirwayAll">
        <property name="toolTip">
         <string>Use both low and high airways for route</string>
        </property>
        <property name="statusTip">
         <string>Use both low and high airways for route</string>
        </property>
        <property name="text">
         <string>&amp;All airways</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">buttonGroup_2</string>
        </attribute>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="Line" name="line_11">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="0,1,1">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRouteCalcDirect">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Calculate a direct flight plan.
Keeps procedures.</string>
        </property>
        <property name="statusTip">
         <string>Calculate a direct flight plan</string>
        </property>
        <property name="text">
         <string>&amp;Direct</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonRouteCalcReverse">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="toolTip">
         <string>Reverse the whole flight plan.
Erases procedures.</string>
        </property>
        <property name="statusTip">
         <string>Reverse whole flight plan</string>
        </property>
        <property name="text">
         <string>&amp;Reverse</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Help</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radioButtonRouteCalcFull</tabstop>
  <tabstop>radioButtonRouteCalcSelection</tabstop>
  <tabstop>spinBoxRouteCalcCruiseAltitude</tabstop>
  <tabstop>pushButtonRouteCalcAdjustAltitude</tabstop>
  <tabstop>radioButtonRouteCalcAirway</tabstop>
  <tabstop>radioButtonRouteCalcAirwayAll</tabstop>
  <tabstop>radioButtonRouteCalcAirwayJet</tabstop>
  <tabstop>radioButtonRouteCalcAirwayVictor</tabstop>
  <tabstop>checkBoxRouteCalcAirwayNoRnav</tabstop>
  <tabstop>checkBoxRouteCalcAirwayTrack</tabstop>
  <tabstop>pushButtonRouteCalcTrackDownload</tabstop>
  <tabstop>horizontalSliderRouteCalcAirwayPref</tabstop>
  <tabstop>radioButtonRouteCalcRadio</tabstop>
  <tabstop>checkBoxRouteCalcRadioNdb</tabstop>
  <tabstop>pushButtonRouteCalcDirect</tabstop>
  <tabstop>pushButtonRouteCalcReverse</tabstop>
 </tabstops>
 <resources>
  <include location="../../littlenavmap.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_2"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
